modules/apps/product-navigation/product-navigation-product-menu-web/src/main/resources/META-INF/resources/js/PageTypeSelector.es.js

```diff
@@ -86,17 +92,37 @@ function PageTypeSelector(props) {
 								{Liferay.Language.get('add-page')}
 							</ClayDropDown.Item>
 						)}
-						<ClayDropDown.Item
-							data-value={Liferay.Language.get('configure')}
-							key={Liferay.Language.get('configure')}
-							onClick={handleOnConfigureClick}
-							title={Liferay.Language.get('configure')}
-						>
-							{Liferay.Language.get('configure')}
-						</ClayDropDown.Item>
+						{props.addCollectionLayoutURL !== '' && (
+							<ClayDropDown.Item
+								data-value={Liferay.Language.get(
+									'add-collection-page'
+								)}
+								key={Liferay.Language.get(
+									'add-collection-page'
+								)}
+								onClick={handleOnAddCollectionPageClick}
+								title={Liferay.Language.get(
+									'add-collection-page'
+								)}
+							>
+								{Liferay.Language.get('add-collection-page')}
+							</ClayDropDown.Item>
+						)}
 					</ClayDropDown.ItemList>
 				</ClayDropDown>
 			</div>
+			<div className="autofit-col ml-2">
+				<ClayLink
+					borderless
+					displayType="unstyled"
+					href="javascript:;"
```

Comment:
If it is a link why not just setting the url here?
`href={props.addCollectionLayoutURL}`

Reply:
👍